release/kubernetes-manifests.yaml

```diff
@@ -741,6 +741,8 @@ spec:
           value: "frontend:80"
         - name: USERS
           value: "10"
+        - name: RATE
```

Comment:
Would need to be added to the kustomize base too:
https://github.com/GoogleCloudPlatform/microservices-demo/blob/main/kustomize/base/loadgenerator.yaml#L84

Reply:
Done